docs/Contributing/adr/0002-github-discussions.md

```diff
@@ -0,0 +1,94 @@
+# ADR-0002: Not using GitHub Discussions 💬
+
+## Status 🚦
+
+Accepted
+
+## Date 📅
+
+2025-07-16
+
+## Context 🔍
+
+Our team faces a challenge with managing complex technical discussions related to GitHub issues:
+
+* 🗣️ GitHub issues are not conducive to having several complex discussions on multiple aspects of a story
+* 💬 These discussions frequently happen in Slack instead
+* ⏰ We lose all history in Slack after 90 days
+* 🔍 There is no way to find the Slack discussion from the issue later
+* 🌐 We are public by default, but Slack discussions are not public
+
+This leads to loss of valuable context and decision-making history that could help future contributors understand why certain choices were made.
+
+We initially considered GitHub Discussions as a potential solution to preserve these conversations and link them to issues.
+
+## Decision ✅
+
+We have decided **not to use GitHub Discussions** for managing complex technical conversations.
+
+This decision was made after evaluating GitHub Discussions and discovering several blocking issues that prevent it from solving our core problems:
+
+* 🔗 **Poor integration with GitHub Issues:** There is no standard linkage between Discussions and Issues, making it difficult to maintain context between the two systems.
+* 🚫 **No Slack notifications:** We cannot get GitHub Discussion notifications in Slack, which is critical for our workflow:
+  * Toast does not support GitHub Discussions
+  * GitHub Slack plugin is too noisy for our needs
+  * We could not find any other suitable integrations
+  * Using a Slack webhook is also too noisy since it cannot DM a person and simply publishes to a channel
+  * Building our own custom integration would be too time-consuming
+
+## Consequences 🎭
+
+### Benefits ✨
+
+* 🎯 Avoids introducing a tool that doesn't solve our core problems
+* ⏱️ Saves time that would be spent on custom integration development
+* 🔄 Prevents workflow fragmentation across too many platforms
+
+### Drawbacks ⚠️
+
+* 📉 We still lose valuable discussion history after 90 days in Slack
+* 🤷 No clear path forward for preserving complex technical discussions
+* 🔍 Future contributors will continue to lack context on decision-making
+
+### Impact 💫
+
+* 🚦 No change to current workflows
+* 🔎 Team will continue searching for a better solution
```

Comment:
Agreed, the drawbacks outweigh the benefits. I think GitHub Discussions is the ideal forum for these conversations (vs adding a new tool/platform), so I'm hopeful that GitHub will continue to develop the feature like they have with Projects.